1d-life/src/1d-life.js

```diff
@@ -41,9 +41,47 @@
   function getNewVal(lifeState, x) {
     // !!! IMPLEMENT ME
 
-    return 0; // instead of this
+    for (let i = 0; i < lifeState.length; i++) {
+
+      let l = x - 1;
+      let r = x + 1;
+
+      if (l < 0) {
+        l = lifeState.length - 1;
+      }
+
+      if (r > lifeState.length - 1) {
+        r = 0
+      }
+
+      if (lifeState[x] === 1) { // if cell is alive and l and r are alive 
```

Comment:
great job!! this can be written a little bit simpler.  The return 1s and 0s can be combined.